securitycenter/assets/main_test.go

```diff
@@ -0,0 +1,86 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Smoke tests for main.go
+
+package main
+
+import (
+	"os"
+	"testing"
+	"time"
+)
+
+func setup(t *testing.T) string {
+	orgId := os.Getenv("GCLOUD_ORGANIZATION")
+	if orgId == "" {
+		t.Skip("GCLOUD_ORGANIZATION not set")
+	}
+	return orgId
+}
+
+func TestAllAssetsReturned(t *testing.T) {
+	org := setup(t)
+	found, err := ListAllAssets(org)
+	if err != nil {
+		t.Errorf("foo failed: %v", err)
+	}
+	if found < 59 {
+		t.Errorf("Not enough results found: %d", found)
+	}
+}
+
+func TestAllProjectsReturned(t *testing.T) {
+	org := setup(t)
+	found, err := ListAllProjectAssets(org)
+	if err != nil {
+		t.Errorf("foo failed: %v", err)
+	}
+	if found > 4 || found == 0 {
+		t.Errorf("Unexpected number of results: %d", found)
+	}
+}
+
+func TestBeforeDateNoAssetsReturned(t *testing.T) {
+	org := setup(t)
+	var nothingInstant = time.Date(2019, 1, 1, 0, 0, 0, 0, time.UTC)
+
+	found, err := ListAllProjectAssetsAtTime(org, nothingInstant)
+	if err != nil {
+		t.Errorf("foo failed: %v", err)
+	}
+	if found > 0 {
+		t.Errorf("Results found for at %v: %d", nothingInstant, found)
```

Comment:
Same comment as above. For these types of failures, please include the expected value. Something like

```
t.Errorf("listAllProjectAssetsAtTime(%s, %v) found $d assets, want 0", org, nothingInstant, found)
```

Reply:
done.